llvm/test/tools/llvm-objcopy/ELF/prefix-symbols.test

```diff
@@ -40,51 +40,11 @@ Symbols:
   - Name:     undef
     Binding:  STB_GLOBAL
 
-# COMMON: Symbols [
-# COMMON-NEXT:  Symbol {
-# COMMON-NEXT:    Name:
-# COMMON-NEXT:    Value: 0x0
-# COMMON-NEXT:    Size: 0
-# COMMON-NEXT:    Binding: Local
-# COMMON-NEXT:    Type: None
-# COMMON-NEXT:    Other: 0
-# COMMON-NEXT:    Section: Undefined
-# COMMON-NEXT:  }
-# COMMON-NEXT:  Symbol {
-# COMMON-NEXT:    Name: foo
-# COMMON-NEXT:    Value: 0x0
-# COMMON-NEXT:    Size: 0
-# COMMON-NEXT:    Binding: Local
-# COMMON-NEXT:    Type: Section
-# COMMON-NEXT:    Other: 0
-# COMMON-NEXT:    Section: .text
-# COMMON-NEXT:  }
-# COMMON-NEXT:  Symbol {
-# BASIC-NEXT:    Name: prefixbar
-# REDEF-NEXT:    Name: prefixbaz
-# COMMON-NEXT:    Value: 0x0
-# COMMON-NEXT:    Size: 0
-# COMMON-NEXT:    Binding: Local
-# COMMON-NEXT:    Type: File
-# COMMON-NEXT:    Other: 0
-# COMMON-NEXT:    Section: .text
-# COMMON-NEXT:  }
-# COMMON-NEXT:  Symbol {
-# COMMON-NEXT:    Name: prefixfoobar
-# COMMON-NEXT:    Value: 0x0
-# COMMON-NEXT:    Size: 0
-# COMMON-NEXT:    Binding: Global
-# COMMON-NEXT:    Type: Function
-# COMMON-NEXT:    Other: 0
-# COMMON-NEXT:    Section: .text
-# COMMON-NEXT:  }
-# COMMON-NEXT:  Symbol {
-# COMMON-NEXT:    Name: prefixundef
-# COMMON-NEXT:    Value: 0x0
-# COMMON-NEXT:    Size: 0
-# COMMON-NEXT:    Binding: Global
-# COMMON-NEXT:    Type: None
-# COMMON-NEXT:    Other: 0
-# COMMON-NEXT:    Section: Undefined
-# COMMON-NEXT:  }
-# COMMON-NEXT:]
+#COMMON:      Symbol table '.symtab' contains 5 entries:
+#COMMON-NEXT:  Num: Value Size Type Bind Vis Ndx Name
+#COMMON-NEXT:   0: {{.*}} 0 NOTYPE  LOCAL  {{.*}}
+#COMMON-NEXT:   1: {{.*}} 0 SECTION LOCAL  {{.*}} foo
```

Comment:
For defined non-ABS symbols, `[[#]]` (which matches a decimal) is slightly better than `{{.*}}`

Reply:
Done.